java/src/test/java/io/process/analytics/tools/bpmn/generator/AppTest.java

```diff
@@ -15,29 +15,74 @@
  */
 package io.process.analytics.tools.bpmn.generator;
 
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 import java.io.File;
 
+import static io.process.analytics.tools.bpmn.generator.internal.FileUtils.fileContent;
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class AppTest {
 
     @Test
+    @Disabled("not implemented yet")
     public void main_generates_xml_output_file() throws Exception {
-        String output = "target/test/output/AppTest/A.2.0_with_diagram.bpmn.xml";
-        generate("src/test/resources/bpmn/A.2.0.bpmn.xml", output);
+        String outputPath = outputPath("A.2.0_with_diagram.bpmn.xml");
+        generateToBpmn(inputPath("A.2.0.bpmn.xml"), outputPath);
 
-        assertThat(new File(output)).exists();
+        File bpmnFile = new File(outputPath);
+        assertThat(bpmnFile).exists().isFile();
         // TODO add xml assertions (assertj and/or xmlunit)
+        assertThat(fileContent(bpmnFile)).contains("BPMN Baby!!");
+    }
+
+    @Test
+    public void main_generates_svg_output_file() throws Exception {
+        String outputPath = outputPath("A.2.0_with_diagram.bpmn.svg");
+        generateToSvg(inputPath("A.2.0.bpmn.xml"), outputPath);
```

Comment:
ℹ️ IMO splitting logic of the App in 2 would help to make theses test more readable:
* Having on one part logic to parse options of the app and read the file given
* Having in another class logic to take the content of the BPMN and outputting the layout.

We should do that at some point.

Reply:
yes, the App class is a mess, it is a little bit better in #25 
i didn't want to work on it for now, but for sure, there is a lot to improve here :smiley_cat:

Reply:
I was just putting my though here for the next steps :). But yep! this can be merged as soon as the other is merged.